content/en/serverless/deployment_tracking/_index.md

```diff
@@ -28,10 +28,19 @@ If you haven’t already, set up the [Amazon Web Services][2] integration first.
 cloudtrail:LookupEvents
 ```
 
+## Troubleshooting 
```

Comment:
I think this section is no longer necessary now, as we just disabled the feature for existing customers to avoid getting more complaints about the cloudtrail noise. However, instead can you help add a sentence in the Setup section to mention that if you already have the permission in your AWS/Datadog IAM role, but you don't see any deployment events populated for any function, then you should reach out to Datadog support to manually enable the feature?

Reply:
Actually, I just made the change myself. @apigirl mind reviewing my proposed change?

Reply:
awesome, thanks @tianchu!